src/test/goal.service.mock.ts

```diff
@@ -0,0 +1,28 @@
+import {Injectable} from "@angular/core"
```

Comment:
This file is also in the wrong directory.
it should be in `src/app/course/_test/_services`